curvefs/src/client/metric/client_metric.h

```diff
@@ -103,6 +103,84 @@ struct MetaServerClientMetric {
           createPartition(prefix, "createPartition") {}
 };
 
+struct LatencyUpdater {
```

Comment:
curvefs/src/common/metric_utils.h already has a LatencyUpdater, you can use it directly

Reply:
fixed